tests/data/config/exclude_match/pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.pydocstringformatter]
+exclude = "**/test_package/**"
```

Comment:
There's no newline at end of file there, maybe we could make the pre-commit exclude for newline and white space wider ?

Reply:
We could although I think stripping white spaces at the end of an docstring would also be a nice feature for us, so we'll need to then turn it off for the sub directory again?

Reply:
I think we should not step on the toes of pre-commit's ``trailing-whitespace`` and ``end-of-file-fixer``. But we can deactivate specifically test files for it like in pylint if we want to handle whitespaces. (https://github.com/PyCQA/pylint/blob/main/.pre-commit-config.yaml#L9)

Reply:
Done with https://github.com/DanielNoord/pydocstringformatter/commit/71534deed2de0cd7254da5fc6699471a3a462d4b